src/XrdPosix/XrdPosixXrootd.hh

```diff
@@ -46,6 +46,7 @@
 
 #include "XrdPosix/XrdPosixOsDep.hh"
 #include "XrdSys/XrdSysPthread.hh"
+#include "XrdPosix/XrdPosixAdmin.hh"
 
 struct XrdOucIOVec;
```

Comment:
And here is a good place to forward declare XrdPosixAdmin

Reply:
done